manifests/cluster-manager/0000_01_addon.open-cluster-management.io_managedclusteraddons.crd.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: managedclusteraddons.addon.open-cluster-management.io
```

Comment:
we should also install clustermanagementaddon crd

Reply:
added